packages/thirdweb/src/extensions/uniswap/read/getUniswapV3Pools.test.ts

```diff
@@ -0,0 +1,48 @@
+import { describe, it, expect, vi, afterEach } from "vitest";
+
+import { getUniswapV3Pools } from "./getUniswapV3Pools.js";
+import {
+  UNISWAPV3_FACTORY_CONTRACT,
+  OHM_CONTRACT_ADDRESS,
+  WETH_CONTRACT_ADDRESS,
+  MOG_CONTRACT_ADDRESS,
+} from "~test/test-contracts.js";
+
+const fetchSpy = vi.spyOn(globalThis, "fetch");
+
+describe("uniswap.getUniswapV3Pool", () => {
+  afterEach(() => {
+    fetchSpy.mockClear();
+  });
+
+  it("should return the WETH/OHM pool address and fee", async () => {
+    const pools = await getUniswapV3Pools({
+      contract: UNISWAPV3_FACTORY_CONTRACT,
```

Comment:
would it make sense to make the contract prop optional and default it to the default uniswap factory?

Reply:
I thought about that, but opted against it because the address can vary on some chains